From 44c574730f920142cb625fff76fd407a8b70dfeb Mon Sep 17 00:00:00 2001
From: Yajun Li <yajunl@codeaurora.org>
Date: Tue, 24 May 2016 21:54:34 +0800
Subject: [PATCH 01/11] drm: Make msm_drm.h uapi header safe for C++

or is a protected keyword in C++ and can't be used at all.

Change-Id: Ie3d2a237291588b6291fca1a49f073a787ae9035
Signed-off-by: Yajun Li <yajunl@codeaurora.org>
---
 include/uapi/drm/msm_drm.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/uapi/drm/msm_drm.h b/include/uapi/drm/msm_drm.h
index 77ba0c34..9ae5074 100644
--- a/include/uapi/drm/msm_drm.h
+++ b/include/uapi/drm/msm_drm.h
@@ -121,7 +121,11 @@ struct drm_msm_gem_cpu_fini {
  */
 struct drm_msm_gem_submit_reloc {
 	__u32 submit_offset;  /* in, offset from submit_bo */
+#ifdef __cplusplus
+	__u32 or_val;
+#else
 	__u32 or;             /* in, value OR'd with result */
+#endif
 	__s32  shift;          /* in, amount of left shift (can be negative) */
 	__u32 reloc_idx;      /* in, index of reloc_bo buffer */
 	__u64 reloc_offset;   /* in, offset from start of reloc_bo */
-- 
2.8.2

